### DISCUSSION TITLE
Default skin template?

### DISCUSSION THREAD

**Discussion by deok420** [ppy/osu#20933] [Q&A]
Will the default lazer skin be released as a template for other people to make skins? Or will there be a skinning wiki for lazer skins, like there is for stable ones (https://osu.ppy.sh/wiki/en/Skinning/osu%21)? I'm asking this, because I used to make a lot of stable skins, but now that I've switched to lazer, there's not a lot to work on. I could also suggest some features be added to the skin editor built in lazer, but that's for a different discussion.

**Comment by smoogipoo** [comment]
There won't be a template for the new default skin because it mostly doesn't use textures/sprites, apart from the approach circle which will probably also not use a texture in the near future.

You'll be able to select the new default skin elements from the in-game skin editor and use those.
